runtime/src/lib.rs

```diff
@@ -127,11 +113,11 @@ pub mod opaque {
 	}
 }
 
-/// The version infromation used to identify this runtime when compiled natively.
```

Comment:
:eyes:

Reply:
Oops, don't know where it went. I'll add it back.